### PR TITLE
Fixing geo-map

### PR DESCRIPTION
**Pull request by SebastianBurke**
modified gruntfile.coffee to include geomap's JS files that are excluded in line 839

### CODE REVIEW

**Comment by SebastianBurke** [comment]
Abandoned the gruntfile modification and brought over the full demo to gcweb

**Comment by Garneauma** [comment]
Pre-approved upon successful testing. This is only a content change.

**Review by Garneauma — CHANGES_REQUESTED** [review]

File: wet-boew/geomap/geomap-en.html [hunk 1]
```diff
@@ -0,0 +1,1557 @@
...
+	"title": "Geomap",
+	"language": "en",
+	"category": "Plugins",
+	"description": "Displays a dynamic map over which information from additional sources can be overlaid.",
+	"tag": "geomap",
+	"parentdir": "geomap",
+	"altLangPrefix": "geomap",
+	"js": ["demo/client_functions"],
+	"dateModified": "2016-04-22"
+}
```
Comment (Garneauma) [inline wet-boew/geomap/geomap-en.html:12]: Please to the following to all 4 HTML files' front-matter:
- Please remove "category", "tag", and "parentDir".
- Rename "altLangPrefix" to "altLangPage" and add the language suffix and file extension, i.e.: "geomap-en.html"

```suggestion
{
	"title": "Geomap",
	"language": "en",
	"description": "Displays a dynamic map over which information from additional sources can be overlaid.",
	"altLangPage": "geomap-en.html",
	"js": ["demo/client_functions"],
	"dateModified": "2016-04-22"
}
```

**Review by Garneauma — CHANGES_REQUESTED** [review]

**Review by Garneauma — CHANGES_REQUESTED** [review]
You did not update geomap-v4.0.30.html (en and fr).

**Review by Garneauma — APPROVED** [review]

**Pull request merged**